docker/configure_workers_and_start.py

```diff
@@ -619,6 +619,8 @@ def main(args: List[str], environ: MutableMapping[str, str]) -> None:
     if not os.path.exists(config_path):
         log("Generating base homeserver config")
         generate_base_homeserver_config()
+    else:
+        log("Homeserver config already exists at: " + config_path)
```

Comment:
Same as above. The commit that made it even says it can be reverted. I mention it in the original comment.
>There are two commits in this pull request that can be reverted. The one is harmless and probably would be good to keep. The other is bad and shouldn't be used long term on Github(tons of log spam):

>* A line is output when Generating base homeserver config that didn't have a mate when config already exists. I added that in before collecting logs while diagnosing the issue. Look for Homeserver config already exists at:.
>* Added the environment variable COMPLEMENT_ALWAYS_PRINT_SERVER_LOGS=1 to the complement CI workflow so when tests gets ran, you won't have to wait for a flake or an error to actually see the log output.

And I agree about extra logging not being here. As a curiousity, is that appropriate in an isolated pull request?

Reply:
You got my meaning exactly. Thanks.